deployment/docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.8'
+
+services:
+  cordra:
+    build:
+      context: cordra
+      dockerfile: docker/Dockerfile
+    env_file:
+      - variables.env
+  tettris_marketplace:
+    container_name: tettris_marketplace
+    restart: always
+    image: public.ecr.aws/dissco/tettris-marketplace:latest
```

Comment:
If we start running on production it would be better to have this fixed to a specific version

Reply:
I will update it to this version (of the PR) after its merged and pushed to AWS